packages/react/src/React.js

```diff
@@ -43,7 +44,9 @@ var React = {
   Component: ReactBaseClasses.Component,
   PureComponent: ReactBaseClasses.PureComponent,
   unstable_AsyncComponent: ReactBaseClasses.AsyncComponent,
-  Fragment: REACT_FRAGMENT_TYPE,
+  Fragment: ReactFeatureFlags.enableReactFragment
```

Comment:
Maybe let's not export it at all? It would be a bit odd to see it on `React` object being undefined.